libs/designsystem/testing-base/src/lib/components/mock.progress-circle.component.ts

```diff
@@ -0,0 +1,22 @@
+import { forwardRef, Component, Input } from '@angular/core';
+
+import { ProgressCircleComponent } from '@kirbydesign/designsystem';

```

Comment:
Add missing ThemeColor import:
```suggestion
import { ProgressCircleComponent, ThemeColor  } from '@kirbydesign/designsystem';

```

Reply:
Not needed. Confused with old mock that has now been removed (previous name).